### PR TITLE
chore: update New Action request Issue Template

### PR DESCRIPTION
**Pull request by jmeridth**
Add checklist of other non-coding steps that should be included with the creation of a new app or action

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_action.yml [hunk 1]
```diff
@@ -12,3 +12,14 @@ body:
         Ex. It would be great to have an action that would ...
     validations:
       required: false
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Items that need to be completed along with the creation of the new App or Action
```
Comment (zkoppert) [inline .github/ISSUE_TEMPLATE/new_action.yml:18]:
```suggestion
      description: Items that need to be completed along with the creation of the new App or Action (to be filled out later when the action is being developed)
```

Comment (zkoppert) [reply .github/ISSUE_TEMPLATE/new_action.yml:18]: Does this add clarity that folks shouldn't think that they need to do these things as perquisites to opening the issue or is there another way to add that clarity?

Comment (zkoppert) [reply .github/ISSUE_TEMPLATE/new_action.yml:18]: or maybe 
```suggestion
      description: Items that should be completed during the development of the new App or Action
```

File: .github/ISSUE_TEMPLATE/new_action.yml [hunk 2]
```diff
@@ -12,3 +12,14 @@ body:
         Ex. It would be great to have an action that would ...
     validations:
       required: false
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Items that need to be completed along with the creation of the new App or Action (to be filled out later when the action is being developed)
+      options:
+        - label: Update [GitHub Actions section of README](https://github.com/github/github-ospo?tab=readme-ov-file#tools-created-by-github-ospo) linking to the new App or Action
+          required: true
```
Comment (zkoppert) [inline .github/ISSUE_TEMPLATE/new_action.yml:21]: Will this require them to check the box before creating the issue?

Comment (jmeridth) [reply .github/ISSUE_TEMPLATE/new_action.yml:21]: Yes. Can make false.

**Review by zkoppert — APPROVED** [review]

**Comment by zkoppert** [comment]
Thanks for getting this in!!!

**Comment by jmeridth** [comment]
> Thanks for getting this in!!!

Your idea. I'm merely the keyboard user. :octocat:

**Pull request merged**